goal_src/jak2/build/all_objs.json

```diff
@@ -0,0 +1,2252 @@
+["hip-door-a-ag-HIPHOG",                       "hip-door-a",                 4,["HIPHOG"], "levels/haven/hiphog"],
+["yellow-barrel-ag",                           "yellow-barrel",              4,["GGA"], "levels/haven/guncourse"],
+["dark-barrel-ag",                             "dark-barrel",                4,["GGA"], "levels/haven/guncourse"],
+["gungame-vis",                                "gungame-vis",                4,["GGA"], "levels/haven/guncourse"],
+["youngsamos-tombd+0-ag",                      "youngsamos-tombd+0",         4,["TOD"], "levels/mars_tomb/entrance"],
+["kid-tombd+0-ag-TOD",                         "kid-tombd+0",                4,["TOD"], "levels/mars_tomb/entrance"],
+["kid-tombd+0-ag-VIN",                         "kid-tombd+0",                4,["VIN"], "levels/power_station"],
+["tpage-1626",                                 "tpage-1626",                 4,["TOD"], "levels/mars_tomb/entrance"],
+["tpage-1627",                                 "tpage-1627",                 4,["TOD"], "levels/mars_tomb/entrance"],
+["tpage-3275",                                 "tpage-3275",                 4,["TOD"], "levels/mars_tomb/entrance"],
+["tombd-vis",                                  "tombd-vis",                  4,["TOD"], "levels/mars_tomb/entrance"],
+["mincan-obs",                                 "mincan-obs",                 3,["MCN"], "levels/no_mans_canyon"],
+["canyon-scenes",                              "canyon-scenes",              3,["MCN"], "levels/no_mans_canyon"],
+["tpage-3341",                                 "tpage-3341",                 4,["MCN"], "levels/no_mans_canyon"],
+["tpage-3343",                                 "tpage-3343",                 4,["MCN"], "levels/no_mans_canyon"],
+["tpage-3342",                                 "tpage-3342",                 4,["MCN"], "levels/no_mans_canyon"],
+["tpage-3344",                                 "tpage-3344",                 4,["MCN"], "levels/no_mans_canyon"],
+["tpage-3471",                                 "tpage-3471",                 4,["MCN"], "levels/no_mans_canyon"],
+["ctypal-baron-statue-break-ag",               "ctypal-baron-statue-break",  4,["MCN"], "levels/no_mans_canyon"],
+["mincan-cogs-ag",                             "mincan-cogs",                4,["MCN"], "levels/no_mans_canyon"],
+["ctypal-break-wall-ag",                       "ctypal-break-wall",          4,["MCN"], "levels/no_mans_canyon"],
+["mincan-lighthouse-ag",                       "mincan-lighthouse",          4,["MCN"], "levels/no_mans_canyon"],
+["water-anim-mincan-ag",                       "water-anim-mincan",          4,["MCN"], "levels/no_mans_canyon"],
+["mincan-lens-ag",                             "mincan-lens",                4,["MCN"], "levels/no_mans_canyon"],
+["mincan-vis",                                 "mincan-vis",                 4,["MCN"], "levels/no_mans_canyon"],
```

Comment:
No clue what `mincan` actually stands for here but we can probably just have this as `levels/canyon/` or `levels/mincan/`